packages/loot-design/src/components/modals/ImportTransactions.js

```diff
@@ -508,6 +538,26 @@ function FieldMappings({ transactions, mappings, onChange, splitMode }) {
   );
 }
 
+function MultipliersField({ multiplierCB, value, onChange }) {
+  const styl = (() => {
+    if (multiplierCB) {
+      return 'inherit';
+    } else {
+      return 'none';
+    }
+  })();
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
  const styl = multiplierCB ? 'inherit' : 'none';
```